api-go/cmd/contributor/main.go

```diff
@@ -130,11 +131,28 @@ func getContributorSVG(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
-	w.Header().Add("content-type", "image/svg+xml;charset=utf-8")
-	w.Header().Add("cache-control", "public, max-age=86400")
 
-	svg = strings.Replace(svg, "%", "%%", -1)
-	fmt.Fprintf(w, svg)
+	if strings.Contains(svg, "svg") {
+		w.Header().Add("content-type", "image/svg+xml;charset=utf-8")
+		w.Header().Add("cache-control", "public, max-age=86400")
+
+		svg = strings.Replace(svg, "%", "%%", -1)
+		fmt.Fprintf(w, svg)
+	} else {
+		w.Header().Add("content-type", "image/png")
```

Comment:
we choose svg but not png before is because svg is relative small (at least 5 times smaller I believe).
Is there any reason to change to png

Reply:
I can only produce `png` when I render the picture in Google Cloud Function.

Reply:
`SVG` is better, let me check the method again, @LiteSun do you have any good suggestions?

Reply:
`svg` would be beteer. see: 
![image](https://user-images.githubusercontent.com/31329157/131459860-fe143ecf-6840-4148-83d4-ba5872992269.png)